.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+dist: xenial
+matrix:
+  include:
+    - python: 2.7
+    - python: 3.5
+    - python: 3.6
+    - python: 3.7
+
+services: xvfb
+
+before_install:
+  - pip install pytest-cov
+  - pip install coveralls
+  - pip install future
+
+install:
+  - python setup.py install
+
+script:
+  - pytest -vs --cov=pysatModelUtils/
```

Comment:
Experimenting with pytest here, but could run nose.  Tests are written the same, just a more streamlined report in the end.

Reply:
Shows as outdated, but we shouldn't stick with nose if it isn't active.

Reply:
I'm open to either.  Testing nose now to see where the issues are.

Reply:
Fixed!